.openshift-ci/crawler/copy-init.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -e
+
+copyAndRunInitScript() {
+    local GCP_VM_NAME="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+        echo "error: missing parameter GCP_VM_NAME" && return 1
+
+    success=false
+    for _ in {1..3}; do
+        if gcloud compute scp /tmp/init.sh "$GCP_VM_NAME:/tmp/init.sh"; then
+            success=true
+            break
+        else
+            echo "Retrying in 5s ..."
+            sleep 5
+        fi
+    done
+
+    if test ! "$success" = "true"; then
```

Comment:
Same as the other nitpick
```suggestion
    if [[[ "$success" != "true" ]]; then
```

Reply:
That's quite a lot of brackets here :)

Reply:
You are right, one too many, I was clearly on a bracket high!